.github/workflows/run_spec_tests.yml

```diff
@@ -0,0 +1,65 @@
+name: Ruby Test Env
+# As opposed to the ci env used before on the Ruby action
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:11.7-alpine
+        env:
+          POSTGRES_USER: test_local_user
+          POSTGRES_PASSWORD: password
+          POSTGRES_DB: i_have_i_need_test
+        ports:
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 2.5.7
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.5.7
+    - name: Ruby gem cache
+      uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gems-
+    - name: Install gems
+      run: |
+        bundle config path vendor/bundle
+        bundle install --jobs 4 --retry 3
+    - name: Setup database
+      env:
+        RAILS_ENV: test
+        POSTGRES_USER: test_local_user
+        POSTGRES_PASSWORD: password
+        POSTGRES_DB: i_have_i_need_test
+        POSTGRES_HOST: localhost
+        POSTGRES_PORT: ${{ job.services.postgres.ports[5432] }}
+        SECRET_KEY_BASE: fcd3d03a727ad50a19d2a89555bc9683
```

Comment:
Is this a secret? If so please alert the team and reset it wherever else we use it now that it's been pushed. If not, maybe we should replace it with a string like `test_secret` so it's clear that it's not sensitive?

Reply:
This secret was generated by me specifically for the test environment. I can't see how it would be used anywhere else. Anyway I've been refining the script and it turns out you don't need any of the environment vars including the secret. (In an earlier version I was trying to use a separate ci environment but that failed). 
So I've removed it.